templates/submission/form/categories.tpl

```diff
@@ -7,7 +7,7 @@
  *
  * Include categories for submissions.
  *}
-{if count($categories)}
+{if $categories && is_array($categories) && count($categories)}
```

Comment:
Aren't the last two checks enough for this case?  Or  is `is_array()` throwing a warning here without the first check?

Reply:
Problem is when `$categories` is `null` which is the cause of this issue . Again PHP now more strict as if a non countable is passed to `count` method, then it throws fatal error . just some extra precaution to make sure anything like previous error does not arise .